History.md

```diff
@@ -1,3 +1,11 @@
+# Version 2.12.0
+
+### Updated
+
+* Changes to support Microsoft Edge browser
+
+Release date: 2016-12-22
+
 # Version 2.11.0

```

Comment:
Usually a one or more changes are combined into a package release by whoever is maintaining the Nuget package. As such it's better to leave this alone and have just the code changes in this PR